pkg/master/master_test.go

```diff
@@ -30,7 +30,7 @@ func TestGetServersToValidate(t *testing.T) {
 	config := Config{}
 	fakeClient := tools.NewFakeEtcdClient(t)
 	fakeClient.Machines = []string{"http://machine1:4001", "http://machine2", "http://machine3:4003"}
-	config.EtcdHelper = tools.EtcdHelper{fakeClient, latest.Codec, nil}
+	config.EtcdHelper = tools.EtcdHelper{fakeClient, latest.Codec, nil, ""}
```

Comment:
IMO it's a must to parameterize our tests for this change, like we did with apiversion. That final "" needs to come from a command line flag, and the test-go.sh file should run it with at least "" and "prefix" values.